Packages/Core/Editor/Generator/GeneratorEditor.cs

```diff
@@ -147,7 +153,11 @@ private void OnEnable()
             };
             generator = generator == null ? new Generator() : generator;
 
+#if UNITY_2018_4_OR_NEWER
```

Comment:
Same as above. The UNITY_2019_1_OR_NEWER condition should be before UNITY_2018_4_OR_NEWER, the second if-statement will never be used when running Unity 2019 since the UNITY_2018_4_OR_NEWER flag will also be true in this case.